lib/turbo-sprockets/parallel_compiler.rb

```diff
@@ -34,7 +34,12 @@ def compile_in_parallel(paths)
 
           { 'files' => {}, 'assets' => {} }.tap do |data|
             manifest.find([path]) do |asset|
-              logger.info("Writing #{asset.digest_path}")
+              if File.exist?(asset.digest_path)
```

Comment:
Can we simplify a bit and do:

```ruby
next if File.exist?(asset.digest_path)
logger.info("Writing #{asset.digest_path}
```

i.e. is there any benefit in writing out the skip message?

Reply:
just debug... i think its safe to skip